go/vt/vtgate/planbuilder/operators/update.go

```diff
@@ -618,7 +638,17 @@ func createFkVerifyOpForChildFKForUpdate(ctx *plancontext.PlanningContext, updSt
 		for _, column := range cFk.ParentColumns {
 			valTuple = append(valTuple, sqlparser.NewColNameWithQualifier(column.String(), parentTbl))
 		}
-		whereCond = sqlparser.AndExpressions(whereCond, sqlparser.NewComparisonExpr(sqlparser.NotInOp, valTuple, sqlparser.ValTuple{updateValues}, nil))
+		var finalExpr sqlparser.Expr = sqlparser.NewComparisonExpr(sqlparser.NotInOp, valTuple, sqlparser.ValTuple{updateValues}, nil)
+		for _, value := range updateValues {
+			finalExpr = &sqlparser.OrExpr{
+				Left: &sqlparser.IsExpr{
+					Left:  value,
+					Right: sqlparser.IsNullOp,
+				},
+				Right: finalExpr,
+			}
+		}
+		whereCond = sqlparser.AndExpressions(whereCond, finalExpr)
```

Comment:
nit: looks like similar changes, can this be extracted to a common method?

Reply:
Fixed 👍